x-pack/plugins/observability_solution/inventory/server/routes/entities/get_identity_fields_per_entity_type.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ENTITY_IDENTITY_FIELDS, ENTITY_TYPE } from '@kbn/observability-shared-plugin/common';
+import { Entity, EntityType } from '../../../common/entities';
+
+export type IdentityFieldsPerEntityType = Map<EntityType, string[]>;
+
+export const getIdentityFieldsPerEntityType = (
+  entities: Array<Pick<Entity, typeof ENTITY_TYPE | typeof ENTITY_IDENTITY_FIELDS>>
```

Comment:
Could it be just?
```suggestion
  entities: Entity[];
```

Reply:
what I was thinking? 🤔 yes I will update

Reply:
https://github.com/elastic/kibana/pull/195250/commits/ca6be0dbd95f3b8bf2835d8464310961c1aa76c5#diff-15651f386471aa18d4fac9f61a7f4512bb16db0bca35db991827824424c7cd99R17